### PR TITLE
Badge GET api

### PR DESCRIPTION
**Pull request by whyDontI**
## Added
- [x] GET /badges

## API Contracts
https://github.com/Real-Dev-Squad/website-api-contracts/pull/29

### CODE REVIEW

**Review by Cartmanishere — COMMENTED** [review]
LGTM

**Review by Rucha1499 — COMMENTED** [review]
Great work!! @whyDontI 🎉 😄 

Can we generate the API Schema for this route?

**Review by Rucha1499 — APPROVED** [review]
Looks great!! 🎉

File: test/integration/badgeController.test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          expect(res).to.have.status(200)
+          expect(res.body).to.be.a('object')
+          expect(res.body.message).to.equal('Badges returned successfully!')
+          expect(res.body.badges).to.be.a('array')
+
+          return done()
+        })
+    })
+  })
+})
```
Comment (swarajpure) [inline test/integration/badgeController.test.js:26]: Where are we adding the badges in the database to `GET` them? 🤔
How will this test case pass on CI? 🤔

**Review by swarajpure — COMMENTED** [review]
Having a doubt in the test case 😅

File: test/integration/badgeController.test.js (hunk 1 above)
Comment (whyDontI) [reply test/integration/badgeController.test.js:26]: Ankush is going to add badges manually in database until we meet the authorization in place.

Comment (swarajpure) [reply test/integration/badgeController.test.js:26]: > 
> 
> Ankush is going to add badges manually in database until we get the authorization in place.

I guess the CI also uses firebase emulators that's why we install it on every commit.
I meant the CI doesn't use the production db, right?
Feel free to correct me 😄

Comment (whyDontI) [reply test/integration/badgeController.test.js:26]: Yes, seems like CI uses emulator, check test.yml file in .git/workflow
CI fetching and adding data in prod DB won't be a good idea

Comment (swarajpure) [reply test/integration/badgeController.test.js:26]: That's what I'm saying 😅 
The prod DB will have the badges details
but the firebase emulator wont and hence the test wouldn't pass 😅 

Let me know if I'm wrong 😄

Comment (whyDontI) [reply test/integration/badgeController.test.js:26]: Oh, that's the convention, even if the GET Endpoint returns an empty array, the response code will still be 200, and the request is considered fulfilled.

Comment (swarajpure) [reply test/integration/badgeController.test.js:26]: Cool

**Review by swarajpure — APPROVED** [review]
Looks good! 👌🏻

**Pull request merged**